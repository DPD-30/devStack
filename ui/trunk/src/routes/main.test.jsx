import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { renderToString } from 'react-dom/server'


import RoutingMain from './main.jsx'

jest.unmock('./main.jsx')


test('RoutingMain matches snapshot', () => {
	const comp = renderToString(<Router><RoutingMain /></Router>)

	expect(comp).toMatchSnapshot()
})
