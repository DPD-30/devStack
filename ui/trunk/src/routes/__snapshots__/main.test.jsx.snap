// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoutingMain matches snapshot 1`] = `""`;
